Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/library/alpine:3.13
+
+ARG network
+ENV IXO_HOME=/home/ixo/
+
+RUN apk add --no-cache ca-certificates libgcc libstdc++ tzdata
+
+COPY bin/ixod /usr/bin/ixod
```

Comment:
This relies on `bin/ixod` binary to be already built - what do you think about including the build process in this Dockerfile? Isn't it the whole point of dockerizing this app? ;)

Reply:
@skolobov one gotcha is of course when we need to get a binary outside Docker, like for using Ledger locally

we need to somehow either build in Docker and output file locally or build in Docker and copy to another image.
maybe we can have 2 dockerfiles? They can override and extend each other if needed

Reply:
I agree that this logic is not obvious. I thought quite a while and decided to do it that way but during the first iteration I actually built needed binary inside of a docker image. 
Now I see that current approach raises questions. I think we need to have one docker file which builds everything inside and if a binary is needed, then we just build in docker as it is now.

Reply:
Updated